### ISSUE TITLE
How can I add LSTM before CRF layer

### ISSUE THREAD

**Issue by zwd13122889** [kyzhouhzau/BERT-NER#77]

**Comment by azmat21** [comment]
you add LSTM in hidden2tag function, then both crf and softmax can use it
`
def hidden2tag(hiddenlayer,numclass):
    bi = tf.keras.layers.Bidirectional(tf.keras.layers.LSTM(100, return_sequences=True))
    linear = tf.keras.layers.Dense(numclass,activation=None)
    hiddenlayer = bi(hiddenlayer)
    hiddenlayer = linear(hiddenlayer)
    return hiddenlayer
`

**Comment by zwd13122889** [comment]
thank you .  In my data , i hava 80 categories , the FBI = 55. How can i improve my accuracy ?

**Comment by linhlt-it-ee** [comment]
Key crf_loss/transition not found in checkpoint
	 [[node save/RestoreV2 (defined at /usr/local/lib/python2.7/dist-packages/tensorflow_estimator/python/estimator/estimator.py:1537) ]]
How can I avoid this error when config "CRF"=True
